config/settings.local.yml

```diff
@@ -0,0 +1,2 @@
+hets:
```

Comment:
I think this was pushed by accident. Please add to `.gitignore`

Reply:
Fixed by commit --amend (that's the only change I made)